Sources/XcodeGenKit/SettingsBuilder.swift

```diff
@@ -202,6 +202,9 @@ extension SettingsPresetFile {
                 symlink.parent() + relativePath,
             ] + possibleSettingsPaths
         }
+        if let moduleResourcePath = Bundle.module.path(forResource: "SettingPresets", ofType: nil) {
+            possibleSettingsPaths.append(Path(moduleResourcePath) + "\(path).yml")
+        }
```

Comment:
@roman-aliyev If @yonaskolb agree with your way, please open PR and do not forget to add this code and copy script SettingPresets to bundle.

Reply:
Also, please add https://github.com/yonaskolb/XcodeGen/pull/1388/files#diff-f913940c58e8744a2af1c68b909bb6383e49007e6c5a12fb03104a9006ae677eR45-R46 and make symbolic link as https://github.com/yonaskolb/XcodeGen/pull/1388/files#diff-9f320f241fc06216ac0b9fed9a549298a11c8f203c299350159cab6946344b14